src/Makefile

```diff
@@ -122,8 +122,9 @@ endif
 # Override default settings if possible
 -include .make-settings
 
-FINAL_CFLAGS=$(STD) $(WARN) $(OPT) $(DEBUG) $(CFLAGS) $(SERVER_CFLAGS)
-FINAL_LDFLAGS=$(LDFLAGS) $(OPT) $(SERVER_LDFLAGS) $(DEBUG)
+# For added compatibility REDIS_CFLAGS and REDIS_LDFLAGS are also supported.
```

Comment:
Instead of this, can we have something that just sets SERVER_CFLAGS to REDIS_CFLAGS if REDIS_CFLAGS is provided by SERVER_FLAGS are not. Seems a bit simpler.